src/components/TaskAccordion.js

```diff
@@ -40,10 +40,9 @@ const TaskAccordion = (props) => {
     const [error, setError] = useState('');
     const toast = useToast();
 
-
+    //saves checked players when page is refreshed
     useEffect(() => {
-        console.log('usingEffect')
-        console.log(task.completedBy);
+        console.log(`usingEffect setCheckedPlayers: ${task.completedBy}`);
         setCheckedPlayers(task.completedBy);
         // eslint-disable-next-line
```

Comment:
Hm totally not related to your code but why is this eslint here? What's the problem on the next line?

Reply:
Resolved